src/main/webapp/script.js

```diff
@@ -5,16 +5,13 @@
  * @author tblanshard
  */
 
-let projectID;
-
 function initBody() {
     fetch('/data').then(console.log('Works'));
-    projectID = "bt-dataflow-sql-demo";
     checkPermissions();
 }
 
 function checkPermissions() {
-    fetch('/check-permissions?projID='+projectID, {method:"GET"})
+    fetch('/check-permissions?projID='+config.projectID+'&saID='+config.serviceAccount, {method:"GET"})
```

Comment:
fetch does a GET request by default, when no options are specified. You could write simplier
```suggestion
    fetch('/check-permissions?projID='+config.projectID+'&saID='+config.serviceAccount)
```

Reply:
Ah good point - I only started putting GET in as well as POST to make things easier to read when looking for different functions but I agree that it's simpler without them :)